Emby.Server.Implementations/Diagnostics/CommonProcess.cs

```diff
@@ -106,25 +106,22 @@ public bool WaitForExit(int timeMs)
             return _process.WaitForExit(timeMs);
         }
 
+        
```

Comment:
Should remove this

Reply:
There seem to be still some references to it, but I'll see if I can change them to the other method without breaking anything

Reply:
Eh I might leave it in anyways. I've got no way of easily testing the TV recorder. But I can quickly test if there's anything to gain by changing this code.
https://github.com/jellyfin/jellyfin/blob/098fcfc5964d71858fd57b0191ddb933c48020ec/MediaBrowser.Api/ApiEntryPoint.cs#L565-L570

Reply:
I just meant the extra line ;)

Reply:
O jeez that makes a lot of difference ;) I'll skip it then because I don't feel like getting into sync issues. Let's just assume all the stop encoding and recording stuff is well threaded.

Reply:
Weird... Visual Studio keeps adding that line

Reply:
done